### DISCUSSION TITLE
Don't logout after updating

### DISCUSSION THREAD

**Discussion by cliffjao** [authelia/authelia#4346] [Q&A] [answered]
Even with Remember Me setting, I get logged out weekly because I have an auto-update cronjob to pull the latest Authelia docker. Is there a way to make it so it doesn't log me out after updating? It even logs me out after `docker restart`.

**Comment by nightah** [comment, marked as answer]
This is likely because you're using the in-memory session provider.
Please read the following:
* https://www.authelia.com/overview/authorization/statelessness/#session-provider
* https://www.authelia.com/configuration/session/redis/

**Reply by EDIflyer** [reply]
Agree - I added `redis` to my Docker setup and it solves this problem really nicely :)
